jdbc/src/java/org/apache/hive/jdbc/HiveResultSetMetaData.java

```diff
@@ -30,6 +30,8 @@
  *
  */
 public class HiveResultSetMetaData implements java.sql.ResultSetMetaData {
+  private static final String DOT = ".";
```

Comment:
It is a bit overloaded to me to create a constant for DOT that is .  - I'm fine with the  . :-) 

cc: @ayushtkn @deniskuzZ Is this a design rule to make this static final? Even in the simplest cases?

Reply:
all constants should be marked as `static final`